docs/landing_generated/index.html

```diff
@@ -7,6 +7,8 @@
     <title>Envoy: C++ L7 proxy and communication bus</title>
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic|Source+Code+Pro">
     <link href="stylesheets/site-00db0e0a.css" rel="stylesheet" />
+    <link rel="shortcut icon" type="image/png" href="/favicon.ico"/>
```

Comment:
Should we fix landing_source also? We are in kind of a bad place here as I have no idea anymore how to even generate landing generated from landing_source. I'm wondering if we should just delete landing_source and live with the generated HTML that we have? That seems bad also. :(